### DISCUSSION TITLE
Please add the ' (apostrophe) serve the same as a , (coma).

### DISCUSSION THREAD

**Discussion by lsholguin** [kazad/instacalc#70] [Ideas]
See, in Mexico, we sometimesuse an apostrophe to denote millions.   For example:   $ 3,500,000.= we (informally) write as $ 3'500.=

Can you please add the apostrophe to behave the same as a coma, so you can write 3'500 and Instacalc ignores it, like it does with the comas?  Or it could (alternatively) work as a 1000X multiplier so, for example, 3'500 would be converted (or treated as) to 3,500,000.

**Comment by kazad** [comment]
Thanks for the suggestion! I have to be careful with `'` because it's used in feet and inches (`3' 6"`), but I have added support for it like this:

https://instacalc.com/57030

`3'500,000 => 3,500,000`

You can use `'` like a comma, but it doesn't automatically go in the millions place: you have to write the rest of the digits. Let's see how this works and then tweak the rules if needed.
